src/app/components/SocialLinks/index.tsx

```diff
@@ -0,0 +1,113 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import getOriginCode from '#app/lib/utilities/imageSrcHelpers/originCode';
+import { Summary } from '#app/models/types/curationData';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import { RequestContext } from '#app/contexts/RequestContext';
+import { createSrcsets } from '#lib/utilities/srcSet';
+import getLocator from '#lib/utilities/imageSrcHelpers/locator';
+import Heading from '../Heading';
+import Image from '../Image';
+import styles from './index.styles';
+
+interface SocialLinksProps {
+  position: number;
+  title: string;
+  summaries: Summary[];
+}
+
+const SocialLinkImage = ({ imageUrl }: { imageUrl: string }) => {
+  const { isLite } = useContext(RequestContext);
+  const DEFAULT_IMAGE_SIZE = styles.IMAGE_SIZE_GROUP_1;
+  const DEFAULT_IMAGE_SIZE_2X = DEFAULT_IMAGE_SIZE * 2;
+
+  const imagePath = imageUrl?.split('{width}')[1];
```

Comment:
Let's use array destructuring:

```suggestion
  const [, imagePath] = imageUrl?.split('{width}');
```